src/components/Icon/IconClear.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import cx from 'classnames';
+import PropTypes from 'prop-types';
+import StyleObjectPropType from '../../prop-types/style';
+import './Icon.scss';
+
+let lastInstanceId = 0;
+
+const propTypes = {
+    className: PropTypes.string,
+    style: StyleObjectPropType(),
+};
+
+const defaultProps = {
+    className: '',
```

Comment:
We should probably use `null` as the default for consistency with the main components. I've already updated IconArrow and IconMore in my PR.